src/navigations/MainNavigation.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import FeedScreen from '~/screens/FeedScreen';
+import SearchScreen from '~/screens/SearchScreen';
+import HotScreen from '~/screens/HotScreen';
+import MyScreen from '~/screens/MyScreen';
+
+const Tab = createBottomTabNavigator();
+
+const MainNavigation: React.FC = () => {
+  let getIconName: Function = (
+    focused: boolean,
+    value: string,
+  ): JSX.Element => (
+    <Ionicons size={25} name={focused ? value : `${value}-outline`} />
+  );
```

Comment:
1. `let` 보다는 `const` 키워드를 쓰는게 맞는 듯 합니다!
2. 아이콘 컴포넌트를 리턴하는 함수이기 때문에 `getIconName` 보다는 `getIcon` 변수명이 맞을 듯 합니다!
3. Typescript에서 리턴 타입이 함수일 때는 Function 보다는 해당 함수의 인자들과 리턴 타입을 명시하는 것을 추천하지만, 이미 인자들의 타입과 리턴 타입이 함수에 나와있기 때문에 함수를 담고 있는 변수에는 리턴 타입을 명시할 필요가 없어보입니다!
4. React 타입 스크립트 파일에서 컴포넌트를 리턴하는 함수들의 리턴 타입을 보통 `React.ReactNode`를 사용하는 것으로 봐왔습니다! `JSX.Element` 대신 `React.ReactNode`를 사용하는 것이 좋아보입니다!

```suggestion
  const getIconName = (
    focused: boolean,
    value: string,
  ): React.ReactNode => (
    <Ionicons size={25} name={focused ? value : `${value}-outline`} />
  );
```

Reply:
말씀하신 부분 반영해서 커밋 푸시했습니다 〰️